vocabularies/UI.xml

```diff
@@ -1448,6 +1448,12 @@ or `204 No Content` if the request included a `Prefer` header with a value of `r
         </Annotation>
       </Term>
 
+      <Term Name="AdaptationHidden" Type="Core.Tag" Nullable="false" AppliesTo="Property EntitySet EntityType">
```

Comment:
```suggestion
      <Term Name="AdaptationHidden" Type="Core.Tag" Nullable="false" DefaultValue="true" AppliesTo="Property EntitySet EntityType">
```
Do we want to allow tagging syntax without explicit value?
~~~xml
<Annotation Term="UI.AdaptationHidden" />
~~~

Reply:
not sure who was faster ... but I committed exactly this change (as I like the short tagging syntax).